server/src/main/java/org/elasticsearch/bootstrap/JNANatives.java

```diff
@@ -260,4 +269,39 @@ static void tryInstallSystemCallFilter(Path tmpFile) {
             logger.warn("unable to install syscall filter: ", e);
         }
     }
+
+    @SuppressForbidden(reason = "need access to fd on FileOutputStream")
+    static void fallocateSnapshotCacheFile(Environment environment, long fileSize) throws IOException {
+        if (Constants.LINUX == false) {
+            logger.debug("not trying to create a shared cache file using fallocate on non-Linux platform");
+            return;
+        }
+        Path cacheFile = SnapshotUtils.findCacheSnapshotCacheFilePath(environment, fileSize);
+        if (cacheFile == null) {
+            throw new IOError(new IOException("Could not find a directory with adequate free space for cache file"));
```

Comment:
why wrap in IOError?

Reply:
Remove that wrapping, sorry for the noise, this was a left-over from some debugging.